test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -501,19 +566,19 @@ def load_config(self, cfg):
         """
         Create a hook configuration file with the provided contents.
         """
-        fn = self.du.create_temp_file(body=cfg)
+        fn = self.du.create_temp_file(hostname=self.serverA, body=cfg)
```

Comment:
Why does this file have to be on serverA? Can't it be from any server? I understand there's no multi-server yet, but using localhost like you did before is probably better.

Reply:
Vincent, on cray host PTL is run from login nodes and not server host and PTL will run all the commands by ssh'ing to server node.